src/token/erc2981/interface.cairo

```diff
@@ -0,0 +1,22 @@
+use openzeppelin::token::erc2981::FeesRatio;
+use starknet::ContractAddress;
+
+
+pub const IERC2981_ID: felt252 = 0x2d3414e45a8700c29f119a54b9f11dca0e29e06ddcb214018fc37340e165ed6;
+
+#[starknet::interface]
+pub trait IERC2981<TState> {
+    fn royalty_info(self: @TState, token_id: u256, sale_price: u256) -> (ContractAddress, u256);
+}
+
+
+#[starknet::interface]
+pub trait IERC2981Setup<TState> {
```

Comment:
This should be provided as internal functions, but since this interface is not part of the standard, we shouldn't provide an external implementation.

Reply:
I have removed this trait and added internal functions matching the ones in Solidity implementation